jmclient/jmclient/client_protocol.py

```diff
@@ -487,9 +487,10 @@ def start_reactor(host, port, factory, ish=True, daemon=False, rs=True, gui=Fals
                 if usessl:
                     reactor.listenSSL(port, dfactory,
                                   ssl.DefaultOpenSSLContextFactory(
-                                      "./ssl/key.pem", "./ssl/cert.pem"))
+                                      "./ssl/key.pem", "./ssl/cert.pem"),
+                                     interface=host)
```

Comment:
IIUC, `interface=host` will make listening bind to localhost, right?
If so then what is the reason to remove it from the SSL listener and not just add it to the non SSL one?  (sorry if this is obvious)

Reply:
The TLS listener does not perform any kind of authentification/authorization. It's just as (un)safe as the non-TLS listener on a public interface.

Reply:
@undeath sorry, my question probably seemed very wierd.  I had inverted colors turned on my browser and the additions looked like removals.  Makes perfect sense to me, cheers.